rasa/utils/log_utils.py

```diff
@@ -43,7 +56,6 @@ def _anonymizer(
         "entity",
         "token_text",
         "user_message",
-        "activity",
```

Comment:
Why is `activity` removed?

Reply:
It was initially added because of the changes in this [PR](https://github.com/RasaHQ/rasa-plus/pull/676/commits/58a7b45a5987bb6afb2ab099b763b8954f636180), but it was reverted. So, it's no longer needed.